src/components/homepage/RecommendProducts.css

```diff
@@ -0,0 +1,79 @@
+/* product-deal */
+
+.deal-section {
+  margin: 1rem;
+  margin-top: 3rem;
+}
+
+.deal-title {
+  text-transform: uppercase;
+  color: var(--secondary-color);
+  font-size: 2rem;
+  letter-spacing: 0.2rem;
+}
+
+.deal-container {
+  /* margin: 2rem; */
+  text-align: center;
+  display: flex;
+  justify-content: center;
+  gap: 1rem;
+  flex-wrap: wrap;
+  /* flex-shrink: 2; */
+}
+
+.product-img {
+  /* width: 13.8rem; */
```

Comment:
Remove commented code

Reply:
yes, actually comments are for later use I am confused their so commented for now.